conduit/plugins/importers/filereader/filereader.go

```diff
@@ -66,14 +66,28 @@ func (r *fileReader) Init(ctx context.Context, _ data.InitProvider, cfg plugins.
 	if r.cfg.FilenamePattern == "" {
 		r.cfg.FilenamePattern = filewriter.FilePattern
 	}
+	r.format, r.gzip, err = filewriter.ParseFilenamePattern(r.cfg.FilenamePattern)
+	if err != nil {
+		return fmt.Errorf("Init() error: %w", err)
+	}
 
 	return nil
 }
 
+// GetGenesis returns the genesis. Is is assumed that
+// the genesis file is available __in addition to__ the round 0 block file.
+// This is because the encoding assumed for the genesis is different
+// from the encoding assumed for blocks.
+// TODO: handle the case of a multipurpose file that contains both encodings.
```

Comment:
Could you elaborate on this TODO in the comment? As it is, I'm not sure exactly what you have in mind.

Also, the canonical genesis file is json. Maybe we should always write it that way?

Reply:
What I mean in the **TODO** is that in principle, it's possible to have a block 0 which includes both the genesis information as well as the block information. But you hint that this may not be such a great idea and we should probably just assume we have a `genesis.json` since that's canonical. Does it actually make sense to have a round 0? Should we eliminate the concept?

Reply:
Plan:
* remove TODO
* file exporter/importer ASSUME genesis.json
* continue with block 0 in appropriate encoding

Reply:
497df82430e75ed109836893133b0bd5d82d1c1f